assets/scss/_custom.scss

```diff
@@ -1928,6 +1928,57 @@ body.td-search {
   color: #ffffff !important;
 }
 
+body.td-home section.case-studies {
+   h2, h3 {
+     text-align: center;
+   }
+  .case-study-list {
+    display: flex;
+    flex-direction: row;
+    max-width: 80vw;
+    margin-left: auto;
+    margin-right: auto;
+    align-items: stretch;
+    gap: clamp(1rem, 4em, 10vw);
+    > .case-study-item {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      width: clamp(6rem, 20%, 50vw);
+      picture, picture img {
+        height: 4.8rem;  
+        text-align: center;
+      }
+      > a {
+        display: block;
+        text-align: right;
+      }
+      padding-bottom: 2em;
```

Comment:
Moving this to an upper level (_list_ instead of the _item_) keeps the same padding below the whole section, yet doesn't add unnecessary paddings when the items are displayed in 2 or 4 rows (it happens on smaller screens).
```suggestion
```